src/utils/mixins.js

```diff
@@ -101,3 +101,95 @@ export const listMixins = {
         }
     }
 }
+
+export const authMixins = {
+    beforeRouteLeave(to, from, next) {
+        for (const prop in this.data) {
+            this.data[prop] = "";
+        }
+        next();
+    },
+    computed: {
+        validations() {
+            const validations = {};
+            for (const key in this.form.data) {
+                validations[key] = this.form.data[key].validations || "";
+            }
+            return validations
+        }
+    },
+    methods: {
+        handleResponse({ data }, isSignup = false) {
+            const auth = isSignup ? data.session : data ;
+
+            Cookies.set("token", auth.token, { expires: new Date(auth.expires), path: "/", domain: process.env.VUE_APP_DOMAIN });
+            this.$store.dispatch("User/setToken", auth.token);
+            if (isSignup) {
+                this.$modal.show("after-signup-wizard")
+            }
+            // TODO: Redirect to ?redirect URL
+            this.$router.push({ name: "dashboard" });
```

Comment:
You don't want to push to the history the login/register/forgot password routes so, replace it with the new route
```suggestion
            this.$router.replace({ name: "dashboard" });
```